webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,53 @@
+import './Avatar.css';
+
+import { Icon, IconVariant, IconSize } from '../index';
+import { Color } from '../../../shared/types';
+
+type AvatarStatus = 'online' | 'offline' | 'playing';
+
+type AvatarProps = {
+  url: string;
+  status?: AvatarStatus;
+};
+
+type LargeAvatarProps = AvatarProps & {
+  edit?: boolean;
+  caption?: string;
+};
+
+export function SmallAvatar({ url, status }: AvatarProps) {
+  return (
+    <div className={`avatar-small__image-wrapper  avatar-status--${status}`}>
+      <img src={url} alt={url} className="avatar-small__image" />
+    </div>
+  );
+}
+
+export function LargeAvatar({
+  url,
+  status,
+  edit = false,
+  caption,
+}: LargeAvatarProps) {
```

Comment:
Better to have to differentiate types of avatars since each of them have different options. This way we don't have to have weird logic to parse them